examples/meta/src/gaussian_process/classifier.sg.in

```diff
@@ -6,20 +6,21 @@ CSVFile f_labels_test("@SHOGUN_DATA@/classifier_4class_2d_linear_labels_test.dat
 #![create_features]
 Features features_train = features(f_feats_train)
 Features features_test = features(f_feats_test)
-MulticlassLabels labels_train(f_labels_train)
-MulticlassLabels labels_test(f_labels_test)
+
```

Comment:
Minor, but avoid adding these spaces (unless the other meta examples have spaces too?)